src/lib/formats/all.cpp

```diff
@@ -1074,6 +1078,9 @@ void mame_formats_full_list(mame_formats_enumerator &en)
 #ifdef HAS_FORMATS_IMG_DSK
 	en.add(FLOPPY_IMG_FORMAT); // img_dsk.h
 #endif
+#ifdef HAS_FORMATS_FS_ISIS
+		en.add(fs::ISIS);
```

Comment:
Only one tab should be used here.

Reply:
ok, I fixed the amount of whitespace. When something like this slips through it usually gets fixed by srcclean before release... anything else to fix, please? Something that doesn't involve just ws?